operators/multiclusterobservability/controllers/multiclusterobservability/multiclusterobservability_controller.go

```diff
@@ -291,6 +291,22 @@ func (r *MultiClusterObservabilityReconciler) Reconcile(ctx context.Context, req
 		}
 	}
 
+	if !rendering.MCOAEnabled(instance) {
+		namespace, labels := renderer.NamespaceAndLabels()
+		toDelete, err := renderer.MCOAResources(namespace, labels)
+		if err != nil {
+			return ctrl.Result{}, err
+		}
+		for _, res := range toDelete {
+			resNS := res.GetNamespace()
+			if err := deployer.Undeploy(ctx, res); err != nil {
+				reqLogger.Error(err, fmt.Sprintf("Failed to undeploy %s %s/%s",
+					res.GetKind(), resNS, res.GetName()))
+				return ctrl.Result{}, err
+			}
+		}
+	}
```

Comment:
Can we flip the logic in general here? Instead of undeploying just skipping deployment in the first place?

Reply:
Not deploying the resources will not remove them from the cluster unless I'm missing something...
This PR does two things when capabilities are not enabled: 
1. Stops MCO from rendering the manifests
2. It will attempt to remove existing deployments of MCOA (to clean up from having it deployed)